### PR TITLE
✨ notification with ntfy

### PR DESCRIPTION
**Pull request by libaurea**
Notifications with ntfy instead of discord. Currently not tested.

### CODE REVIEW

**Comment by libaurea** [comment]
Works :)
image

**Comment by libaurea** [comment]
An example in mobile view with the latest changes.
11yTB8hc3st6

**Comment by libaurea** [comment]
It would also be possible to include [message priorities](https://docs.ntfy.sh/publish/#message-priority) and [authentication](https://docs.ntfy.sh/publish/#authentication).

The reverse direction becomes more exciting. Either Lighthouse would have to subscribe to the feed itself or be accessible via its own webhook url.

**Review by I-Al-Istannen — APPROVED** [review]

**Comment by libaurea** [comment]
Example interaction
image

File: src/main/java/de/ialistannen/lighthouse/cli/CliArguments.java [hunk 1]
```diff
@@ -66,10 +66,13 @@ public interface CliArguments {
   @Option(names = "--bot-channel-id", description = "The channel id the bot should send updates to")
   Optional<String> botChannelId();
 
-  @Parameter(index = 0, description = "Discord webhook URL or discord bot token", paramLabel = "URL|TOKEN")
+  @Option(names = "--ntfy", description = "Use ntfy to send notifications and receive update requests. Default: false")
+  boolean ntfy();
+
+  @Parameter(index = 0, description = "Webhook URL or discord bot token", paramLabel = "URL|TOKEN")
   String webhookUrlOrToken();
```
Comment (I-Al-Istannen) [inline src/main/java/de/ialistannen/lighthouse/cli/CliArguments.java:73]: This can now also take a ntfy webhook URL, right? Can you edit the description?

**Review by I-Al-Istannen — CHANGES_REQUESTED** [review]

**Review by I-Al-Istannen — APPROVED** [review]

**Pull request merged**